tests/run/cpp_class_extern_overloaded_traits.pyx

```diff
@@ -0,0 +1,49 @@
+# tag: cpp
+
+from libcpp cimport bool as bool_t
+
+cdef:
+	struct S1:
+		int i
+
+	struct S2:
+		int i
+
+	extern from "cpp_source.h":
```

Comment:
This also doesn't need a separate test file. You can use verbatimly inlined C code.
```suggestion
	extern from *:
	    """
	    #include <stdio.h>
        #include <map>

        unsigned int i;

        void test(std::map<unsigned int, unsigned int> m) {
	        printf("Map size: %d\n", m.size());
        }

        class testclass {
        public:
	        void a() {
		        printf("testclass->a();\n");
	        }
	        void a(unsigned int i) {
		        printf("testclass->a(%d);\n", i);
	        }
        };
	    """
```